<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>180</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>470</width>
    <height>180</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>470</width>
    <height>180</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Frequency Data</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>451</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>8</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <property name="verticalSpacing">
         <number>18</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="TLC_label">
          <property name="text">
           <string>Total letter count:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="totalLetterCount"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="MCL_label">
          <property name="text">
           <string>Most common plaintext letter(s):</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="mostCommonLetter"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="EFV_label">
          <property name="text">
           <string>Expected frequency chi-square:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="expectedFrequencyChiSquare"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="standardButtons">
         <set>QDialogButtonBox::Close</set>
        </property>
        <property name="centerButtons">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="separator">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="expectedFreqTable">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <row>
       <property name="text">
        <string>A</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>B</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>C</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>D</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>E</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>F</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>G</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>H</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>I</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>J</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>K</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>L</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>M</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>N</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>O</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>P</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Q</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>R</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>S</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>T</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>U</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>V</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>W</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>X</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Z</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>Expected %</string>
       </property>
      </column>
      <item row="0" column="0">
       <property name="text">
        <string/>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>145</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
